rclpy/package.xml

```diff
@@ -49,8 +51,16 @@
 
   <doc_depend>python3-sphinx</doc_depend>
   <doc_depend>python3-sphinx-rtd-theme</doc_depend>
+  <!-- We add these modules as doc_depends in order to include them into the
+  autodoc_mock_imports list. This prevents TypeErrors from being thrown
+  when these modules import constants from rclpy C modules.
+  TODO: Remove these depends once a better solution is found.-->
+  <doc_depend>rclpy.duration</doc_depend>
+  <doc_depend>rclpy.qos</doc_depend>
+  <doc_depend>rclpy.clock</doc_depend>
```

Comment:
Is there any discussion about this anywhere? I'd like to understand it better, because it seems... like an abuse of the dependency system.

Reply:
Not apart from what was mentioned in the PR description. I agree we're taking advantage of the `doc_depend` tag but I couldn't find a better way to deal with the import errors. I'm all ears for a better solution!